src/qibolab/_core/pulses/pulse.py

```diff
@@ -19,15 +20,35 @@
     "VirtualZ",
 ]
 
-PulseId = int
+PulseId = UUID1
```

Comment:
I think this change breaks all acquisition on qblox because of this line: https://github.com/qiboteam/qibolab/blob/85169192a5f1eea4e5d35654fcd1ab2e9b51663e/src/qibolab/_core/instruments/qblox/results.py#L113
which fails to cast the `UUID1` to `int`. I have not investigated in detail, however reverting the commits related to the uuid changes fixes the issue, that's why I think it is related.

If we would like to keep the uuid (even though I am not sure of the main motivation), most likely it should be simple to fix by dropping the cast to `int`. However, we should probably also make sure it does not break other parts of the code, such as the QM driver.